5.x/crud-fluent-syntax.md

```diff
@@ -184,6 +184,20 @@ CRUD::field('products')
         ->tab('Others');
 ```
 
+<a name="fluent-fields-conditionable-methods"></a>
+### Conditionable methods
+
+Sometimes you may need a method to be chained on a field based on another condition. Backpack takes advantage of <a href="https://laravel.com/docs/10.x/queries#conditional-clauses" target="_blank">Laravel Conditionable</a> to easily let you accomplish that. You can either use ```->when()``` or ```unless()```:
+
+```php
+CRUD::field('email')
+    ->when($condition, function ($crud) {
+        $crud->unless($anothercondition, function ($crud) {
+            $crud->hint('My conditioned hint');
+        })
+    });
+```
```

Comment:
Hmmm I think this example might be slightly misleading. 

1) What gets passed as a parameter to the function? Is the the `$crud` or is it actually the `$field`?!
2) Does it make more sense for us to teach people to use `->unless()` inside the closure... or outside it?

Please take another look at this @maurohmartinez - we want this example to be as clear as day.

Reply:
I simplified it and renamed the `$crud` param to `$field`.